lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -3628,6 +3628,334 @@ class ConvertSymConstrainRangeOp
 };
 } // namespace
 
+namespace {
+class ConvertOnnxVariantAtenRotaryEmbeddingOp
+    : public OpConversionPattern<OnnxVariantAtenRotaryEmbeddingOp> {
+public:
+  using OpConversionPattern::OpConversionPattern;
+
+private:
+  struct RotaryParameters {
+    int64_t batchSize;
+    int64_t sequenceLength;
+    int64_t hiddenSize;
+    int64_t headSize;
+    int64_t rotaryEmbeddingDim;
+    int64_t numHeads;
+    int64_t maxSequenceLength;
+    int64_t positionIdsFormat;
```

Comment:
in onnx, the int position_ids_format;   // Format of position ids - 0 is (1), 1 is (batch_size, sequence_length)
 But in current implementation, we only assume  position_ids : (batch_size, sequence_length), the position_ids_format  is not used, so we can delete it.